chef/data_bags/crowbar/migrate/neutron/030_fix_rename_l3_role.rb

```diff
@@ -0,0 +1,17 @@
+def upgrade ta, td, a, d
+  if d.fetch('elements_expanded', {}).has_key? 'neutron-l3'
```

Comment:
`if d.fetch('elements_expanded', {}).key? 'neutron-l3'` that's what hound would suggest.